sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineOperationsTests.java

```diff
@@ -1258,6 +1260,51 @@ public void canOperateVirtualMachine() {
         Assertions.assertEquals(PowerState.DEALLOCATED, vm.powerState());
     }
 
+    @Test
+    public void canCreateVirtualMachineWithEphemeralOSDisk() {
+        VirtualMachine vm = computeManager.virtualMachines()
+            .define(vmName)
+            .withRegion(Region.US_WEST3)
+            .withNewResourceGroup(rgName)
+            .withNewPrimaryNetwork("10.0.0.0/28")
+            .withPrimaryPrivateIPAddressDynamic()
+            .withoutPrimaryPublicIPAddress()
+            .withPopularLinuxImage(KnownLinuxVirtualMachineImage.UBUNTU_SERVER_18_04_LTS)
+            .withRootUsername("Foo12")
+            .withSsh(sshPublicKey())
+            .withSize(VirtualMachineSizeTypes.STANDARD_D8S_V3)
```

Comment:
Is it possible to use a smaller VM size (D8 seems 8-core?) for test, or this is already the smallest?

Reply:
Sure... I'll change it to smaller size.

Reply:
Changed.